### PR TITLE
Full table for crm tap

### PR DESCRIPTION
**Pull request by JingLin0**

### CODE REVIEW

File: tap_tool/sync.py [hunk 1]
```diff
@@ -109,5 +103,5 @@ def sync(
         )
     elif integration == "pipedrive":
         return pipedrive(
-            company, api_key, replication_method, output, discover, bigquery, catalog,
```
Comment (paddie) [inline tap_tool/sync.py]: so, no incremental option?

File: tap_tool/utils.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def terminate(signum, frame):
         new_state = json.loads(line)
     new_state_fp = commit_config(**new_state)
     # push state
-    if not tap == "tap-dynamics":
+    if not replication_method == "FULL_TABLE":
```
Comment (paddie) [inline tap_tool/utils.py]: Wouldn't that simply mean that it was in fact incremental?

```python
if replication_method == "INCREMENTAL":
    write(bq_client, company, new_state_fp.name, stream_id)
```

File: tap_tool/sync.py [hunk 3]
```diff
@@ -109,5 +103,5 @@ def sync(
...
+            company, api_key, "FULL_TABLE", output, discover, bigquery, catalog,
```
Comment (paddie) [inline tap_tool/sync.py]: so, no incremental option?

File: tap_tool/utils.py (hunk 2 above)
Comment (JingLin0) [reply tap_tool/utils.py]: good idea

File: tap_tool/sync.py (hunk 3 above)
Comment (JingLin0) [reply tap_tool/sync.py]: just re-test syncing pipedrive data, my mistake, it can be incremental.

Comment (paddie) [reply tap_tool/sync.py]: that makes a lot of sense 👍

**Pull request merged**